src/SourceBuild/content/test/Microsoft.DotNet.SourceBuild.SmokeTests/SourceBuiltArtifactsTests.cs

```diff
@@ -61,9 +65,4 @@ public void VerifyVersionFile()
             Directory.Delete(outputDir, recursive: true);
         }
     }
-
-    private void ExtractFileFromTarball(string tarballPath, string filePath, string outputDir)
-    {
-        ExecuteHelper.ExecuteProcessValidateExitCode("tar", $"--wildcards -xzf {tarballPath} -C {outputDir} {filePath}", OutputHelper);
```

Comment:
There are other spots in the test project that call `tar` (OmniSharpTests, SdkContentTests). Should those be updated too?

Reply:
I initially didn't because they were slightly different usages that the new method I added wouldn't work for.  I have since added the required logic to remove all `tar` usage.